AsyncDisplayKit/Details/UIView+ASConvenience.h

```diff
@@ -42,6 +42,7 @@ NS_ASSUME_NONNULL_BEGIN
 
 - (void)setNeedsDisplay;
 - (void)setNeedsLayout;
```

Comment:
Agree, let's remove `layoutIfNeeded` for now. It actually does not add anything extra for us at the moment.